test/client/fixtures/automation/content-editable/api-actions-content-editable-test/index-test.js

```diff
@@ -439,6 +439,9 @@ $(document).ready(function () {
         var fixedText             = 'Test' + String.fromCharCode(160) + 'me' + String.fromCharCode(160) + 'all!';
         var inputEventRaisedCount = 0;
 
+        // NOTE IE11 does not raise input event on contenteditable element
+        var expectedInputEventRaisedCount = !browserUtils.isIE11 ? 12 : 0;
```

Comment:
It's preferred to use strait condition where it's possible 
`var expectedInputEventRaisedCount = browserUtils.isIE11 ? 0 : 12;`